templates/docs/examples/patterns/image/container/aspect-ratio/2-3.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image Container / Aspect Ratio / 2/3{% endblock %}
```

Comment:
I wonder if we need a separate example for every single aspect ratio.

There is probably little value for it, as we likely don't want to embedd them separately in the docs. They are quite self explanatory, so having one of them as an example, with a listing of other available class names should be enough.

Obviously for the sake of reference and testing we should have a single "Image container / All aspect ratios" example that would just showcase all of them all together?

Reply:
Revised to use two examples: [16:9](https://vanilla-framework-5097.demos.haus/docs/examples/patterns/image/container/aspect-ratio/16-9?theme=light) (embedded in [documentation](https://vanilla-framework-5097.demos.haus/docs/patterns/images#image-container-with-aspect-ratio) as this is the most frequent use case) and [all](https://vanilla-framework-5097.demos.haus/docs/examples/patterns/image/container/aspect-ratio/all?theme=light)